### PR TITLE
Updates to oep-0025-proc-incremental-improvements.rst

### PR DESCRIPTION
**Pull request by michellephilbrick**
Looking for reviews/feedback on the TL;DR section added at the beginning!

### CODE REVIEW

File: oeps/oep-0025-proc-incremental-improvements.rst [hunk 1]
```diff
@@ -26,6 +26,23 @@ OEP-25: Incremental Improvements
 
 .. _Original pull request: https://github.com/edx/open-edx-proposals/pull/66#pullrequestreview-151034037
 
+TL;DR
+=====
```
Comment (nedbat) [inline oeps/oep-0025-proc-incremental-improvements.rst:30]: Very minor nit: headings read better in the file if you leave a blank line after them.

File: oeps/oep-0025-proc-incremental-improvements.rst [hunk 2]
```diff
@@ -26,6 +26,23 @@ OEP-25: Incremental Improvements
...
+* **JIRA can help coordinate this effort:**  `The Incremental Improvements (INCR) JIRA project`_ allows for a standardized ticket workflow to ensure task requirements and goals are explicitly outlined for contributors to pick up and go.
+* **Quality over quantity:** Not all project tasks will fit into the INCR workflow specifications, but those that are identified as appropriate can be entered into the ticket flow for completion by contributors. To avoid dilution of effort, tickets should be limited to a modest amount at any given time and agreed upon prior to creation via an Open edX forum (such as a team Slack channel).
+
+.. _The Incremental Improvements (INCR) JIRA project: https://openedx.atlassian.net/jira/software/c/projects/INCR/issues/INCR-570?filter=allopenissues
+
+High-Level INCR Workflow Process
+--------------------------------
+* Tickets must meet the requirements for the INCR Jira project outlined in the `Specification`_ section of this document
```
Comment (nedbat) [inline oeps/oep-0025-proc-incremental-improvements.rst:40]: These bullets should end with a period.

File: oeps/oep-0025-proc-incremental-improvements.rst [hunk 3]
```diff
@@ -26,6 +26,23 @@ OEP-25: Incremental Improvements
...
+* INCR tickets should only be written for the parts of the project for which they are appropriate (those that can realistically be broken into smaller tasks and don’t require more extensive knowledge of existing code
```
Comment (nedbat) [inline oeps/oep-0025-proc-incremental-improvements.rst:41]: This is missing a closing paren.

File: oeps/oep-0025-proc-incremental-improvements.rst [hunk 4]
```diff
@@ -26,6 +26,23 @@ OEP-25: Incremental Improvements
...
+* Work on INCR tickets can be done by (but not limited to): Developers (experienced with Open edX or wanting more experience, computer programming students, hackathon teams, new edX technical employees, etc.
```
Comment (nedbat) [inline oeps/oep-0025-proc-incremental-improvements.rst:42]: Was this meant to be a longer list? It only has "Developers" in it now.  It's also missing a closing paren.

**Review by nedbat — CHANGES_REQUESTED** [review]

**Comment by nasthagiri** [comment]
Congratulations on your first PR!

**Pull request merged**